### PR TITLE
TFW-1624 - add conditional showing of GDPR notice / styles

### PR DESCRIPTION
**Pull request by tVeratti**

### CODE REVIEW

File: components/auth/signIn.js [hunk 1]
```diff
@@ -4,11 +4,15 @@ import styles from './signIn.scss';
 import { Logo, Input, Button, Notification } from '../index';
 import formValidation, { removeError } from '../util/formValidation';
 
+const GDPR_AGREE = 'GDPR_AGREE';
```
Comment (troyblank) [inline components/auth/signIn.js]: So, we going to use this style for all local store keys?

File: components/auth/signIn.scss [hunk 2]
```diff
@@ -19,10 +19,20 @@
 }
 
 .sign-in__notice {
-    display: none;
-    margin-top: 2.5rem;
-    color: $color-grey-dark;
+    margin: 2.5rem -5rem -2rem;
```
Comment (troyblank) [inline components/auth/signIn.scss]: any chance these can be added as spacing vars?

File: components/auth/signIn.js (hunk 1 above)
Comment (tVeratti) [reply components/auth/signIn.js]: Only if you like it. Always up for discussion.

File: components/auth/signIn.js [hunk 3]
```diff
@@ -41,6 +45,7 @@ export default class SignIn extends PureComponent {
         if (hasErrors) {
             this.setState({ errors });
         } else {
+            window.localStorage.setItem(GDPR_AGREE, 'true');
```
Comment (troyblank) [inline components/auth/signIn.js]: This seems a bit annoying for the user, I wish we could disable this forever, but alas this is not a signed in page. I can't think of a way of making it go away for good.

Comment (tVeratti) [reply components/auth/signIn.js]: I'm sure we could if we created an api endpoint that stored this flag in the db... but that might be somewhat overkill.

File: components/auth/signIn.scss (hunk 2 above)
Comment (tVeratti) [reply components/auth/signIn.scss]: Yeah - maybe they should also fit our 4px multiplier size thingy thang?

File: components/auth/signIn.js (hunk 3 above)
Comment (tVeratti) [reply components/auth/signIn.js]: Good point - and anything we'd use to track them wouldn't be any better than a fresh browser storage.

File: components/auth/signIn.scss (hunk 2 above)
Comment (tVeratti) [reply components/auth/signIn.scss]: Ok so I added new spacing var and slightly tweaked the overall style to match our base 4px sizes.

**Pull request merged**